app/javascript/shared/components/tags.jsx

```diff
@@ -48,7 +48,8 @@ class Tags extends Component {
   }
```

Comment:
adding this notice here for the whole file, the components is lacking `propTypes validation`